lib/esaebsad/plugins/commands/help.rb

```diff
@@ -9,12 +11,15 @@ class Help < ESAEBSADCommand
   set :prefix, /^@@/
   match "help", method: :list
   match /help (.+)/, method: :advanced
-  
+
   def list(msg)
-    msg.reply "List of commands: #{$help_commands.keys.join(', ')}"
+    commands = Variables.get_help.keys.join(', ')
+    msg.reply "List of commands: #{commands}"
   end
-  
+
   def advanced(msg, command)
-    msg.reply($help_commands.keys.include?(command) ? $help_commands[command] : "Command not found.")
+    help = Variables.get_help
+    message = help.include?(command) ? help[command] : 'Command not found.'
```

Comment:
Normal double quotations ("text") is preferred in my projects. I'll fix it tho.

(TODO: create some sort of style guide)